kolibri/plugins/learn/assets/src/routes/index.js

```diff
@@ -51,39 +49,42 @@ export default [
   {
     name: PageNames.ROOT,
     path: '/',
-    handler: () => {
-      if (get(isUserLoggedIn) && get(channels).length) {
-        return router.replace({ name: PageNames.HOME });
+    redirect: () => {
```

Comment:
is this changing because the `next` is added anove in the unassigned content guard rather than rhe `router.replace`?